### PR TITLE
Rep ssl

### PR DESCRIPTION
**Pull request by akshatsikarwar**
/silent

### CODE REVIEW

**Review by roborivers — CHANGES_REQUESTED** [review]
<p><a href='https://bloomberg.github.io/comdb2/contrib.html#code-style'>Coding style</a> check: <b>Error</b>. &#x26A0;.<br>Smoke testing: Success &#10003;.<br>Cbuild submission: Success &#10003;.<br>Regression testing: <b>7/554 tests failed</b> &#x26A0;.</p><p>The first 10 failing tests are: <br>sc_resume_logicalsc_generated<br>sc_timepart<br>schemalk<br>incremental_backup_usenames<br>online_compaction<br>truncatesc_offline_generated<br>truncatesc_ufid_on_generated</p>

**Review by roborivers — CHANGES_REQUESTED** [review]
<p><a href='https://bloomberg.github.io/comdb2/contrib.html#code-style'>Coding style</a> check: <b>Error</b>. &#x26A0;.<br>Smoke testing: Success &#10003;.<br>Cbuild submission: Success &#10003;.<br>Regression testing: <b>7/556 tests failed</b> &#x26A0;.</p><p>The first 10 failing tests are: <br>selectv_rcode_disable_svonly_nop_generated<br>sc_transactional_rowlocks_generated<br>online_compaction<br>timepart_trunc<br>basic_ssl_generated<br>sc_downgrade<br>truncatesc_offline_generated</p>

**Review by roborivers — CHANGES_REQUESTED** [review]
<p><a href='https://bloomberg.github.io/comdb2/contrib.html#code-style'>Coding style</a> check: <b>Error</b>. &#x26A0;.<br>Smoke testing: Success &#10003;.<br>Cbuild submission: Success &#10003;.<br>Regression testing: <b>6/557 tests failed</b> &#x26A0;.</p><p>The first 10 failing tests are: <br>sc_resume_logicalsc_generated<br>sc_timepart<br>sicountbug<br>sc_lotsoftables<br>sc_lotsoftables_logicalsc_generated<br>truncatesc_offline_generated</p>

File: tests/opt/bb/tmp_testdir_generated.test/lrl.options
[the remark this answers is not in the document]
Comment (akshatsikarwar) [reply tests/opt/bb/tmp_testdir_generated.test/lrl.options:1]: Done!

**Review by roborivers — CHANGES_REQUESTED** [review]
<p><a href='https://bloomberg.github.io/comdb2/contrib.html#code-style'>Coding style</a> check: <b>Error</b>. &#x26A0;.<br>Smoke testing: Success &#10003;.<br>Cbuild submission: Success &#10003;.<br>Regression testing: <b>6/559 tests failed</b> &#x26A0;.</p><p>The first 10 failing tests are: <br>sc_timepart_logicalsc_generated<br>sc_timepart<br>sc_resume<br>dbcreate<br>sc_lotsoftables_logicalsc_generated<br>truncatesc_offline_generated</p>

**Review by riverszhang89 — APPROVED** [review]
Very surgical - good to merge!

**Pull request merged**